src/module/skb/bpf/skb_hook.bpf.c

```diff
@@ -111,6 +111,7 @@ struct skb_data_ref_event {
 static __always_inline int process_skb_l2_l4(struct retis_context *ctx,
 					     struct retis_raw_event *event,
 					     struct skb_config *cfg,
+					     u64 sections,
```

Comment:
Whats the benefit of this? Does it compensate making the functions uglier and more confusing?